Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -467,6 +588,115 @@ def get_users_command(client):
     )
 
 
+def get_users_preference_command(client):
+    result = client.get_users_preference()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    del result['success']
+    table_header = list(result.keys())
+    display_title = "User's Preference"
+    markdown = tableToMarkdown(display_title, result, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.User.Preference',
+        outputs=result
+    )
+
+
+def get_logpoints_command(client):
+    result = client.get_logpoints()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "LogPoints"
+    allowed_loginspects = result.get('allowed_loginspects')
+    if allowed_loginspects and len(allowed_loginspects) > 0:
+        table_header = list(allowed_loginspects[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_loginspects, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.LogPoints',
+        outputs_key_field='ip',
+        outputs=allowed_loginspects
+    )
+
+
+def get_repos_command(client):
+    result = client.get_repos()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "LogPoint Repos"
+    allowed_repos = result.get('allowed_repos')
+    if allowed_repos and len(allowed_repos) > 0:
+        table_header = list(allowed_repos[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_repos, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Repos',
+        outputs_key_field='repo',
+        outputs=allowed_repos
+    )
+
+
+def get_devices_command(client):
+    result = client.get_devices()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    table_header = []
+    display_title = "Devices"
+    allowed_devices = result.get('allowed_devices')
+    if allowed_devices and len(allowed_devices) > 0:
+        table_header = list(allowed_devices[0].keys())
+    markdown = tableToMarkdown(display_title, allowed_devices, headers=table_header)
```

Comment:
![image](https://user-images.githubusercontent.com/46249224/119656009-de2c0080-be32-11eb-876f-4d804e16faf9.png)

Please update the context in this command to `LogPoint. Device.Address` and `LogPoint. Device.OperationSystem`
for example:
{
    "LogPoint": {
        "Devices": [
            {
                "Address": "127.0.0.1/192.168.1.20",
                "OperationSystem": "Windows Server"
            }
        ]
    }
}

Do the same for the table

Reply:
This request is adjusted in [this](https://github.com/demisto/content/pull/12790/commits/6a3409918ccd6e1e8772ebec198cd99325b59f28) commit.